ming_sim/context.py

```
@@ -239,11 +239,48 @@ def character_context(character: Character) -> str:
+    "新机构专办": "新机构专办",
+}
+
+
+def held_authority_context(
+    character: Character, db: GameDB, *, turn: Optional[int] = None,
+) -> str:
+    """大臣可见的在持授权定性摘要；裸 authority_records.id 永不入面。"""
+    if turn is None:
+        try:
+            turn = int(db.load_state().turn)
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)